pandas/src/period.pyx

```diff
@@ -1150,6 +1150,31 @@ cdef class Period(object):
         return period_format(self.ordinal, base, fmt)
 
 
+class Period(_Period):
+    """
+    Represents an period of time
+
+    Parameters
+    ----------
+    value : Period or compat.string_types, default None
+        The time period represented (e.g., '4Q2005')
+    freq : str, default None
+        One of pandas period strings or corresponding objects
+    year : int, default None
+    month : int, default 1
+    quarter : int, default None
+    day : int, default 1
+    hour : int, default 0
+    minute : int, default 0
+    second : int, default 0
+    """
+
+    def __radd__(self, other):
+        # __radd__ on cython extension types like _Period is not used, so
```

Comment:
ahh so this means you can leave Period and we don't need _Period. That's only necessary for Timestamp for example because we are extending a c class itself
here the class is already in cython

but what it does mean is that __add__ needs to handle reversed args directly (and similarly for other arith ops)